tests/ingestion.py

```diff
@@ -0,0 +1,115 @@
+import requests
+import json
+import pystac
+from pystac import STACValidationError
+import os
+import boto3
+
+
+class StacIngestion:
+
+    """Class representing various test operations"""
+
+    def __init__(self):
+        self.ingestor_url = os.getenv("INGESTOR_URL")
+        self.stac_url = "https://stac.test.maap-project.org"
+        self.collections_endpoint = "/collections"
+        self.items_endpoint = "/ingestions"
+        self.current_file_path = os.path.dirname(os.path.abspath(__file__))
+
+    def validate_collection(self, collection):
+        try:
+            pystac.validation.validate_dict(collection)
+        except STACValidationError:
+            raise STACValidationError("Validation failed for the collection")
+
+    def validate_item(self, item):
+        try:
+            pystac.validation.validate_dict(item)
+        except STACValidationError:
+            raise STACValidationError("Validation failed for the item")
+
+    def get_authentication_token(self):
+        # Authentication - Get secret value
+        stac_register_service_id = os.getenv("STAC_REGISTER_SERVICE_ID")
+        stack_name = f"MAAP-STAC-auth-dev"
+
+        # session = boto3.session.Session()
+        client = boto3.client("secretsmanager", region_name=os.getenv("REGION_NAME"))
+        secret_id = f"{stack_name}/{stac_register_service_id}"
```

Comment:
see my comment above, we should simplfy and have the secret_id being an environment variable instead of having `STAC_REGISTER_SERVICE_ID` and hard coded `stack_name`.